tests/test_md.py

```diff
@@ -7,11 +7,21 @@
 import ase.io
 from ase.build import bulk
 from ase.calculators.emt import EMT
+from wfl.autoparallelize import autoparainfo
 
 from wfl.generate import md
 from wfl.configset import ConfigSet, OutputSpec
+from wfl.generate.md.abort import AbortOnCollision
 
 
+def select_every_10_fs_for_tests(traj):
+    return [at for at in traj if at.info["MD_time_fs"] % 10 == 0]
```

Comment:
I think `MD_time_fs` is a float, so you probably shouldn't use `% 10 == 0` with it. Not that I can think of a safe way to do this that isn't pretty complicated (save the time of last call, do some rounding, etc).

Reply:
True, I didn't think about this too much because it's supposed to be just a simple test (so we know exactly how it behaves) and not a user-facing user-proof function. For what it's worth `10.5 % 10 = 0.5` so this particular example works ok...

Reply:
Yeah, it's just bad practice. What do you think about passing the MD step #, an integer, instead of using the time, for the example?

Reply:
I added a new info["MD_step"] and used it in a revised version of the test.

Reply:
of course, thanks for the solution.